_includes/setup/get-sdk.md

```diff
@@ -1,23 +1,34 @@
-## Get the Flutter SDK
-
-To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path.
-Running `flutter doctor` shows any remaining dependencies you may need to install.
-
-### Clone the repo
-
-If this is the first time you're installing Flutter on this machine, clone the
-`beta` branch of the repository and then add the `flutter` tool to your path:
-
-{% commandline %}
-git clone -b beta https://github.com/flutter/flutter.git
-export PATH=`pwd`/flutter/bin:$PATH
-{% endcommandline %}
+## Get the Flutter SDK {#get-sdk}
+
+1. Download the following installation bundle to get the latest beta release of the
+Flutter SDK (for other release channels, and older builds, see the [SDK
+archive](/sdk-archive/) page.):
+{% if page.url contains "linux" %}
+    * [(loading...)](#){:.download-latest-link-linux}
+{% else %}
+    * [(loading...)](#){:.download-latest-link-macos}
+{% endif %}<br><br>
+1. Extract the file in the desired location, e.g.:
```

Comment:
For macOS, you might say something like "Either double click to extract the archive, or extract the file in the desired location from the command line:"

In reality, you could probably double-click on the archive in Linux too, since most distros have a built-in extractor, but I'm not sure how universal that is (probably pretty universal).